src/ucs/datastruct/list.c

```diff
@@ -0,0 +1,78 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2024. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "list.h"
+
+#include <ucs/debug/log.h>
+#include <ucs/debug/memtrack_int.h>
+#include <ucs/sys/math.h>
+
+void ucs_list_print_links(ucs_list_link_t *head, unsigned int num_nodes)
```

Comment:
Can `head` be `const`?

Reply:
`print_links` -> `print_nodes` or simply `print`

Reply:
changed to nodes because it doesn't print the elements

Reply:
and head is now const